toolbox/gpu_operator.py

```diff
@@ -211,6 +211,46 @@ def to_y(_s):
 
         return PlaybookRun("gpu_operator_bundle_from_commit", opts)
 
+    @staticmethod
+    def build_push_image(
```

Comment:
What does this have to do with the GPU operator? It seems pretty generic

Reply:
It doesn't specifically. I only had it included with GPU operator as it's intended use-case was with it. Unsure if it will need anything GPU operator specific though, and if not I will likely abstract it. Will ask @kpouget about it in our meeting tomorrow morning.